### DISCUSSION TITLE
求激活

### DISCUSSION THREAD

**Discussion by user2027** [LFTEC/LearningFucker#42] [General]
请将下面代码复制到Github讨论区中，收到激活码后，调用activate命令进行激活
------------------------------License------------------------------
puK68+Pv2qazOTZzg54TYjuwk8Jkt7Qu7XDcw8yJz57zn5n9LQz2LAGjPnNUOaUpAVD
0HZJ3HDO3u45xXoRh3TxrzUIlXno82tueSuCyvF+XNiGdgepGCxnRM84ZH4HE/ksLGe
Wl86KKOxutNms+X9ISWj5fENy6AUlETm06XJcs3vwSpr7Fq4eHQ1+O5/Wd4CwNnqqWT
z708OLNRq3nsbkd0qrh6+LjUz21d/5edywWWVySmsL8HeTF34jKcUi73OE/5svj0iWx
cygV+6wTq9iqPapbPepcrv+0R8xVsixfdpZGW6aMF4nfgmLWqB+cADEk+G8SMhnwspl
7/w8pvO27LS7TtgehG/SdFZCZZiHs7NhqgP6X0Xpp0dlM6kP434AEyVxFGQV74Kckuj
BRnrxOAe6/wmWiRKkd7ywhsA+8QMVr8eeb3V5arXEbX2TNBwF0TW4tREbPhE4A/Nbbm
RBlP92iAJl8l2i+8usoflRrF+kzzFN9Bq5XJg6iw4P7QZGfv9MyaCkz6gYPZ5xRbZem
QauFpIUAv0JXlgdetJ8RF5drYY/XPxFlLS7EGjp/tzeesSGsQoXP/w0b2NenQddCCM8
e/hD3R9VhQvjhZVyQLLvG3rtaVCfT7kPmkIrA9qbrYCB5FZmc/PbwThfx+vULXsWQDG
GHCDFvNjoqicRLeOpjqtI+7J0KtKGSLAK0lsZpPgUdh3+sBkJIWDQilvJmiDk+2FoA5
DYO0lDwmK74PCKNOkDzI5jX6kOTLtoRSSdWX/AUAhC7zLE/tHF/TVz2is35C76YKBmv
D66wEoN898hD+QZwOqdDHZcAJR3qg2/SN0V4bX0RwYLwmb4EKzvpEH/3IETkT1UeSDl
+WagXOQ5MMLHOhmWO9ivbzukedAMR6DTZT1Ga2trrtxoHcESkTn9FWHHlhth3+pVI9O
7bgYFBSkOkBvqEprg9OuTC+49kBhdZ8p+uL2cBYqkXZDzFNDy+uFEwWOjdNdC9ELAZh
Hxz4vGGUaPNHAp3glV+PrtfOMIDFxLedJa4ZGGnFdtKh2cuWMmKXVSn6+qgqMCmXmTX
90hy94ataBjhGxsCfiOah2rtAeOo0THB4bP3z6u1BGqf2lbNRNZLz5rOyhcmqUf1CP0
363E7Vrr5vYtAgOYJ97DERsgy7RM3zMujRXhs3+p+GFcbEeCddycxgt3jCchZDgThm8
2SGro7/wmyCPEg75qhih+Eih52w/Qop/9PFzyyXBGXaZWvkO5uebdzSjce5xN9BgeoC
KwSwKtC898ZSQlO+o953TaJpby/iccjRPJwZjzCFrmtUBbFPYkzhCKuwjhn7TyXzYaV
fy5mrI2xoREvB44SwP0liRsagQ==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
D84eGCwdVBS/vC94CW1Jys0bclzLW5fbyZ02a46dbn9iG9DNw3g9isvzdZTkjAAymKQ4cl2o/L2oqftwSZFXLwqYPj5cwk1O37WMVc+o2eg00bUQikpK+LcoMkMY8offEjiE5e5dDWCbqzdZxnjzHUasyR97aElqQKY+QkDQ+pkYTRq/J+kF/mT9HsEX/cra3uzHeSfFTrTYTDMunicMRKAVN0DDTM0Z3V6rJ9ZtbjZXV5JZPgEHR9qgg3JKNaR+LeSoDg0P7U4pIv6/P1ut2ahBrwq1ALtLYnazAcGIwtfm/Jj75zqpFUIByI7SGUrejY8BAa564h4u6kOoiqWU2C+oi3niTjDEpZ4JatWHTNEBSuoe7rLLblHBgzR5WTePkYkYKM8XOYtE5wqn6blMLivTs5qL+ywQZC+8eVgTOCX1jxat4QpD+8Wfmdq31qQx/QiBclvvfOZembTi1FVOPI+79jTqZ4dGeYmgnnYLC6bY3Rn8pNOx00KZ8lM4sMca4xkeE8Z2zj3A77RflCxuG6j6duSkt/+70pEocyDafqlRiwHPEQtXWmMFXfVBqXJ/Nd+rt1F91Ms1TTtOLxL247gU0IyyetZpD9wndhaplzpuB2bQFJv6Xt7uMwgQ2PRMEReCDwNPwNWrxdz5QzgYWWNvXpAjb6WQvanVyUtkhRF8HeZQTtzWPGBaEPQimtLhEl9cb3F6yCpi3WmPka3MM/amAIIdtbQI+YmI1GihcUKvlcW6tCb4UFJ3fzIzmo4YNprgR0EV7peFo4Y81HX5EkoFQH5nriFDtAUjaHbgzePrzyxJMkLKnsTUJFKhx16Fgj+b06TSsc/nEZNg+O/tQCmPEb9AfuFI5t5LCHO8zVB7fqSxiRyusI8+GzmzH1AmzAx5TnFZrKYm62lTTy3x5wlyr9zSXDBTc+SkBC1zK+D5dSni5pZa3dOl3dm+lsr/tNNRa9kG2IqsNdapvzvR3HJNbuJjTGn69crSnrSANpC484d5GHWdD/iLM6q2KfWmcTnATERVCe26OZaZdomy46FOtAciDtLRMwLoFI9bpdyWXs6+3cqRmeP7wuUf8ZDhiUV0OgiWTMOV7EbeA/7i6Yq3qTF+tqHJ2ldwyerQ8rk5F8yRNswZjM5ShUY/iwHiToF5xhzKqXPpLY24tNKDvI4MSc4oL6HiW1AsBV42w4Oe+GYp1nGbJntErlduBivaYRraoSoiNUSIV+cfVze0x86thqjm/p/NKsTteDNpc5Qb8IH02d2r1Gz0iNRQsQhTFH/qeqQAv0DMLLof4qaqnGi0a000fFal4vT2pUM5elWZmA4UYtH7zx+atIO9oPTX5pZMB3dlBriShBNN5QuEKQ==
